### ISSUE TITLE
Remove unused `requirements.txt` file

### ISSUE THREAD

**Issue by stevenkolawole** [chaoss/grimoirelab-sortinghat#502]
Since Poetry is being used for installing dependencies, can I go ahead and remove the unused `requirements.txt` file?

**cc**: @sduenas @vchrombie

**Comment by vchrombie** [comment]
Hi @SteveKola, thanks for opening the issue.

I'm not sure if we are ready for this at the very moment. Poetry is not just for installing but we are planning to use it for managing the releases and packaging too. We have many things to be done before we make a release of this tool. We are defining how the releases will be generated in GrimoireLab. We still have work pending in this area.

In any case, I would wait for @sduenas decision on this.

**Comment by SteveKola** [comment]
> I'm not sure if we are ready for this at the very moment. Poetry is not just for installing but we are planning to use it for managing the releases and packaging too. We have many things to be done before we make a release of this tool. We are defining how the releases will be generated in GrimoireLab. We still have work pending in this area.

Alright. I was just thinking that everything that we'd possibly need `requirements.txt` (in conjunction with pip) for, is already covered for by `poetry.lock` so there's a bit of redundancy there.

**Comment by sduenas** [comment]
As @vchrombie said, there are a few steps we need to complete before we can remove the file. Let's keep this issue opened and as soon as we have everything ready, we can remove it.

**Comment by sduenas** [comment]
@vchrombie @jjmerchante do you think we can remove this file?

**Comment by sduenas** [comment]
This is completed.
